guides/v2.1/extension-dev-guide/routing.md

```diff
@@ -1,107 +1,185 @@
 
-class Create extends \Magento\Framework\App\Action\Action
-{
-    public function execute()
-    {
-        $this->_forward('noroute');
-    }
-}
-</pre>
+* `%name%` - The unique name of your router in Magento.
+* `%classpath%` - The path to your router class.    
+    Example: [`Magento\Robots\Controller\Router`]
+* `%sortorder%` - The sort order of this entry in the router list. 
 
 
-<h3>Routing processing</h3>
+## `routes.xml`
 
-Routing is processed in the following way:
+The `routes.xml` file maps which module to use for a URL with a specific `frontName` and area.
+The location of the `routes.xml` file in a module, either `etc/frontend` or `etc/adminhtml`, specifies where those routes are active.
 
-* Modules provide information on their routers through the `routerList` parameter of `Magento\Framework\App\RouterList` type in `di.xml`.
-* `FrontController` obtains active routers and checks whether a request can be processed.
-* If a request cannot be processed by any router, the default router is used.
-* If a request can be processed by a router, the router finds a route with matching `frontName` and looks through corresponding modules. If a {% glossarytooltip c1e4242b-1f1a-44c3-9d72-1d5b1435e142 %}module{% endglossarytooltip %} has matching controller and action names, a router instantiates this controller.
+The content of this file uses the following format:
 
-The `dispatch()` method of the `Magento\Framework\App\Action\Action` class requests an instance and returns its response.
+``` xml
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:App/etc/routes.xsd">
+    <router id="%routerId%">
+        <route id="%routeId%" frontName="%frontName%">
+            <module name="%moduleName%"/>
+        </route>
+    </router>
+</config>
+```
 
-For this class, the `Magento\Framework\App\ActionInterface` processes the requests through its actions. Also, the following classes participate in processing the requests:
+Where:
 
-* The <a href="{{ site.mage2000url }}lib/internal/Magento/Framework/App/State.php" target="_blank">Magento\Framework\App\State</a>  class provides information on the state of the application, that is, current mode, installation date, and so on.
-* The <a href="{{ site.mage2000url }}lib/internal/Magento/Framework/App/AreaList.php" target="_blank">Magento\Framework\App\Arealist</a> class serves to configure the application areas through the `di.xml` file
-* The <a href="{{ site.mage2000url }}lib/internal/Magento/Framework/App/Area/FrontNameResolverInterface.php" target="_blank">Magento\App\Area\FrontNameResolverInterface</a> class resolves the dynamic area's front names
+* `%routerId` - The name of the router in Magento.    
+    See the reference tables in the [Router class section].
+* `%routeId%` - The unique node id for this route in Magento.
+* `%frontName%` - The first segment after the base URL of a request.
+* `%moduleName%` - The name of your module.
 
```

Comment:
Either make this list consistent with the other list above that uses a pattern of "specifies the..." or make the other list consistent with this one. :)